v63002/instream.csh

```diff
@@ -0,0 +1,44 @@
+#!/usr/local/bin/tcsh -f
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+#-------------------------------------------------------------------------------------
+# List of subtests of the form "subtestname [author] description"
+#-------------------------------------------------------------------------------------
+# gtm5178	    [vinay] Tests YDB reports BLKTOODEEP errors as warnings
+#-------------------------------------------------------------------------------------
+
+echo "v63002 test starts..."
+
+# List the subtests separated by spaces under the appropriate environment variable name
+setenv subtest_list_common     ""
+setenv subtest_list_non_replic "gtm5178"
+setenv subtest_list_replic     ""
+
+
+if ($?test_replic == 1) then
+	setenv subtest_list "$subtest_list_common $subtest_list_replic"
+else
+	setenv subtest_list "$subtest_list_common $subtest_list_non_replic"
+endif
+
+# Use $subtest_exclude_list to remove subtests that are to be disabled on a particular host or OS
+setenv subtest_exclude_list	""
+# Filter out white box tests that cannot run in pro
```

Comment:
Same comments as another pull request. Lines 36-39 can be removed. com/SUITE changes are needed etc.